certified-connectors/SmartDialog/apiDefinition.swagger.json

```diff
@@ -1,808 +1,1376 @@
 {
-  "swagger": "2.0",
-  "info": {
-    "version": "1.0",
-    "title": "SmartDialog",
-    "description": "SmartDialog is a platform used to send and receive SMS- and WhatsApp messages. It offers a wide variety of smart messaging services.",
-    "contact": {
-      "name": "Arena Interactive Support",
-      "email": "support@arenainteractive.fi",
-      "url": "https://www.arenainteractive.fi/contact"
-    }
-  },
-  "host": "api.arena.fi",
-  "basePath": "/api-integrations/v1",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [],
-  "produces": [],
-  "paths": {
-    "/customers": {
-      "get": {
-        "responses": {
-          "200": {
-            "description": "default",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "customerUsers": {
-                  "type": "array",
-                  "items": {
-                    "type": "object",
-                    "properties": {
-                      "customerId": {
+    "swagger": "2.0",
+    "info": {
```

Comment:
@robin-larsson Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.